internal/target/pulp_target.go

```diff
@@ -0,0 +1,33 @@
+package target
+
+const TargetNamePulpOSTree TargetName = "org.osbuild.pulp.ostree"
+
+type PulpOSTreeTargetOptions struct {
+	// ServerAddress for the pulp instance
+	ServerAddress string `json:"server_address"`
+
+	// Repository to import the ostree commit to
+	Repository string `json:"repository"`
+
+	// BasePath for distributing the repository (if new)
+	BasePath string `json:"basepath,omitempty"`
+
+	Username string `json:"username"`
+	Password string `json:"password"`
```

Comment:
Shouldn't these be marked as `omitempty`, since credentials can be set also in the worker configuration?

Reply:
Done.